service/gitlab/gitlab_mr_discussion.go

```diff
@@ -117,6 +121,10 @@ func (g *GitLabMergeRequestDiscussionCommenter) postCommentsForEach(ctx context.
 			}
 			_, _, err := g.cli.Discussions.CreateMergeRequestDiscussion(g.projects, g.pr, discussion)
 			if err != nil {
+				if err, ok := err.(*gitlab.ErrorResponse); ok {
+					discussionJson, _ := json.MarshalIndent(discussion, "", "  ")
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
var discussionJson should be discussionJSON